azurerm/internal/services/netapp/netapp_volume_resource.go

```diff
@@ -194,6 +194,43 @@ func resourceNetAppVolume() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+
+			"data_protection_replication": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				ForceNew: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"endpoint_type": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Default:  "dst",
+							ValidateFunc: validation.StringInSlice([]string{
+								"dst",
+							}, true),
+						},
+
+						"remote_volume_location": azure.SchemaLocation(),
+
+						"remote_volume_resource_id": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ValidateFunc: azure.ValidateResourceID,
+						},
+
+						"replication_schedule": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"_10minutely",
```

Comment:
this is a fun value, maybe we should change it to
```suggestion
								"10minutes",
```

and inject the odd SDK value behind the scenes?

Reply:
Completely agree with the oddness of the value, I'll work on the translation, trying to be consistent with our tooling where customers will still see _10minutelly